NavigateU/NavigateU/Coordinator/FlowCoordinator.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  FlowCoordinator.swift
+//  NavigateU
+//
+//  Created by Faki Doosuur Doris on 06.04.2024.
+//
+
+import Foundation
+import UIKit
+
+protocol SomeFlowCoordinator: AnyObject {
+    func toRegisterationScreen()
+    func toLoginScreen()
+    func toHomeScreen()
+}
+
+final class FlowCoordinator: SomeFlowCoordinator {
+
+    private let rootViewController: UINavigationController
+    private let tabBarController = TabBarViewController()
+
+    init(rootViewController: UINavigationController) {
+        self.rootViewController = rootViewController
+    }
+    func toRegisterationScreen() {
+        let registrationViewController = RegistrationScreenViewController()
+        rootViewController.pushViewController(registrationViewController, animated: true)
+    }
+    func toLoginScreen() {
+        let loginViewController = LoginViewController()
+        rootViewController.pushViewController(loginViewController, animated: true)
```

Comment:
We prefer to use ModuleBuilder entity for creating all of out modules, like i used MainModuleBuilder in my example, that i've sent you